Makefile

```
@@ -2,13 +2,13 @@
 # BlockAuth Package — Development Commands
 # ============================================================================
 
-.PHONY: format lint check install
+.PHONY: format lint check install test build clean typecheck help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)